js/packages/navigation/stacks.tsx

```diff
@@ -126,22 +126,40 @@ const DeepLinkBridge: React.FC = React.memo(() => {
 	return null
 })
 
-let Components: typeof RawComponents
+let Components: {
+	Chat: typeof Chat
+	Settings: typeof Settings
+	Onboarding: typeof Onboarding
+	Account: typeof Account | any
```

Comment:
Why `| any` ?
Please if you can find a way without `any` types 👍 Just `typeof Account` doesn't works ?